public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,239 @@
+          description: 'A successful balance request response'
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/GetBalanceOk'
+                  - example:
+                      balance: 12345678.90
+              example:
+                _links:
+                  self:
+                    href: /customs/guarantees/23GB0123456789AB2/balance
+                balance: 12345678.90
+        '400':
+          description: 'Bad Request'
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/BadRequest'
+                  - description: The GRN or supplied Json payload were invalid
+                    example:
+                      code: BAD_REQUEST
+                      message: The provided Json was malformed (it should only contain one field named accessCode)
+        '404':
+          description: 'Not Found'
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/NotFound'
+                  - description: The GRN or access code did not match an existing guarantee
+                    example:
+                      code: NOT_FOUND
+                      message: Guarantee balance not found
+        '406':
+          description: 'Not Acceptable'
+          content:
+            application/json:
+              schema:
+                allOf:
+                  - $ref: '#/components/schemas/errorResponse'
+                  - example:
+                      code: ACCEPT_HEADER_INVALID
+              examples:
+                badRequest:
+                  description: The accept header is missing or invalid
+                  value:
+                    code: ACCEPT_HEADER_INVALID
+      deprecated: false
+      security:
+        - userRestricted:
+            - common-transit-convention-guarantee-balance
+components:
+  schemas:
+    errorResponse:
+      title: errorResponse
+      required:
+        - code
+      type: object
+      properties:
+        code:
+          type: string
+    hal-document-schema.json:
+      title: hal-document-schema.json
+      type: object
+      properties:
+        _links:
+          allOf:
+            - $ref: '#/components/schemas/Links'
+            - description: An object containing links to other resources
+        _embedded:
+          type: object
+          additionalProperties: {}
+          description: An object containing embedded resources
+      description: A HAL document
+    CustomsGuaranteesBalancesrequest:
+      title: CustomsGuaranteesBalancesrequest
+      required:
+        - accessCode
+      type: object
+      properties:
+        accessCode:
+          maxLength: 4
+          minLength: 4
+          type: string
+          description: The access code granting access to make use of the guarantee.
```

Comment:
Wording is slightly awkward but for v1.0 I wasn't able to suggest an alternative. 

Suggestion:
The `accessCode` granting access to make use of the guarantee.